prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```

Comment:
`flex-basis` only works when it is applied to children of elements with `display: flex;` try moving all of your `flex-basis` properties to the rule set for `.card-wrapper`.